SLURM_JOBID:  119802
SLURM_ARRAY_JOB_ID:  119796
SLURM_ARRAY_TASK_ID:  24
Thu Oct  3 09:30:15 CEST 2019
[34m--- Run analysis ---[0m
matlab -nodesktop -softwareopengl -nosplash -nodisplay < /network/lustre/iss01/home/zoe.brusselle/calc_best_param_model.m

                            < M A T L A B (R) >
                  Copyright 1984-2017 The MathWorks, Inc.
                   R2017b (9.3.0.713579) 64-bit (glnxa64)
                             September 14, 2017

 
To get started, type one of these: helpwin, helpdesk, or demo.
For product information, visit www.mathworks.com.
 
>> >> >> [Warning: patID:24] 
>> >> >> >> >> Starting parallel pool (parpool) using the 'local' profile ...
[Warning: The cluster failed to cancel the job execution. The error was: Unable
to read file
'/network/lustre/iss01/home/zoe.brusselle/.matlab/local_cluster_jobs/R2017b/Job21.in.mat'.
No such file or directory.] 
[> In parallel.internal.cluster.CJSJobMethods.cancelOneJob (line 53)
  In parallel.job.CJSConcurrentJob/cancelOneJob (line 53)
  In parallel.Job/cancel (line 1242)
  In parallel.Cluster/hDeleteOneJob (line 906)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster reported an error while deleting an unavailable job.  This
job may already have been deleted.] 
[> In parallel.internal.cluster.CJSJobMethods.destroyOneJob (line 77)
  In parallel.job.CJSConcurrentJob/destroyOneJob (line 48)
  In parallel.Job/delete (line 1176)
  In parallel.Cluster/hDeleteOneJob (line 921)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster failed to cancel the job execution. The error was: Unable
to read file
'/network/lustre/iss01/home/zoe.brusselle/.matlab/local_cluster_jobs/R2017b/Job22.in.mat'.
No such file or directory.] 
[> In parallel.internal.cluster.CJSJobMethods.cancelOneJob (line 53)
  In parallel.job.CJSConcurrentJob/cancelOneJob (line 53)
  In parallel.Job/cancel (line 1242)
  In parallel.Cluster/hDeleteOneJob (line 906)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster reported an error while deleting an unavailable job.  This
job may already have been deleted.] 
[> In parallel.internal.cluster.CJSJobMethods.destroyOneJob (line 77)
  In parallel.job.CJSConcurrentJob/destroyOneJob (line 48)
  In parallel.Job/delete (line 1176)
  In parallel.Cluster/hDeleteOneJob (line 921)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster failed to cancel the job execution. The error was: Unable
to read file
'/network/lustre/iss01/home/zoe.brusselle/.matlab/local_cluster_jobs/R2017b/Job23.in.mat'.
No such file or directory.] 
[> In parallel.internal.cluster.CJSJobMethods.cancelOneJob (line 53)
  In parallel.job.CJSConcurrentJob/cancelOneJob (line 53)
  In parallel.Job/cancel (line 1242)
  In parallel.Cluster/hDeleteOneJob (line 906)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster reported an error while deleting an unavailable job.  This
job may already have been deleted.] 
[> In parallel.internal.cluster.CJSJobMethods.destroyOneJob (line 77)
  In parallel.job.CJSConcurrentJob/destroyOneJob (line 48)
  In parallel.Job/delete (line 1176)
  In parallel.Cluster/hDeleteOneJob (line 921)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster failed to cancel the job execution. The error was: Unable
to read file
'/network/lustre/iss01/home/zoe.brusselle/.matlab/local_cluster_jobs/R2017b/Job24.in.mat'.
No such file or directory.] 
[> In parallel.internal.cluster.CJSJobMethods.cancelOneJob (line 53)
  In parallel.job.CJSConcurrentJob/cancelOneJob (line 53)
  In parallel.Job/cancel (line 1242)
  In parallel.Cluster/hDeleteOneJob (line 906)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster reported an error while deleting an unavailable job.  This
job may already have been deleted.] 
[> In parallel.internal.cluster.CJSJobMethods.destroyOneJob (line 77)
  In parallel.job.CJSConcurrentJob/destroyOneJob (line 48)
  In parallel.Job/delete (line 1176)
  In parallel.Cluster/hDeleteOneJob (line 921)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster failed to cancel the job execution. The error was: Unable
to read file
'/network/lustre/iss01/home/zoe.brusselle/.matlab/local_cluster_jobs/R2017b/Job25.in.mat'.
No such file or directory.] 
[> In parallel.internal.cluster.CJSJobMethods.cancelOneJob (line 53)
  In parallel.job.CJSConcurrentJob/cancelOneJob (line 53)
  In parallel.Job/cancel (line 1242)
  In parallel.Cluster/hDeleteOneJob (line 906)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster reported an error while deleting an unavailable job.  This
job may already have been deleted.] 
[> In parallel.internal.cluster.CJSJobMethods.destroyOneJob (line 77)
  In parallel.job.CJSConcurrentJob/destroyOneJob (line 48)
  In parallel.Job/delete (line 1176)
  In parallel.Cluster/hDeleteOneJob (line 921)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster failed to cancel the job execution. The error was: Unable
to read file
'/network/lustre/iss01/home/zoe.brusselle/.matlab/local_cluster_jobs/R2017b/Job26.in.mat'.
No such file or directory.] 
[> In parallel.internal.cluster.CJSJobMethods.cancelOneJob (line 53)
  In parallel.job.CJSConcurrentJob/cancelOneJob (line 53)
  In parallel.Job/cancel (line 1242)
  In parallel.Cluster/hDeleteOneJob (line 906)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster reported an error while deleting an unavailable job.  This
job may already have been deleted.] 
[> In parallel.internal.cluster.CJSJobMethods.destroyOneJob (line 77)
  In parallel.job.CJSConcurrentJob/destroyOneJob (line 48)
  In parallel.Job/delete (line 1176)
  In parallel.Cluster/hDeleteOneJob (line 921)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster failed to cancel the job execution. The error was: Unable
to read file
'/network/lustre/iss01/home/zoe.brusselle/.matlab/local_cluster_jobs/R2017b/Job27.in.mat'.
No such file or directory.] 
[> In parallel.internal.cluster.CJSJobMethods.cancelOneJob (line 53)
  In parallel.job.CJSConcurrentJob/cancelOneJob (line 53)
  In parallel.Job/cancel (line 1242)
  In parallel.Cluster/hDeleteOneJob (line 906)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster reported an error while deleting an unavailable job.  This
job may already have been deleted.] 
[> In parallel.internal.cluster.CJSJobMethods.destroyOneJob (line 77)
  In parallel.job.CJSConcurrentJob/destroyOneJob (line 48)
  In parallel.Job/delete (line 1176)
  In parallel.Cluster/hDeleteOneJob (line 921)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster failed to cancel the job execution. The error was: Unable
to read file
'/network/lustre/iss01/home/zoe.brusselle/.matlab/local_cluster_jobs/R2017b/Job28.in.mat'.
No such file or directory.] 
[> In parallel.internal.cluster.CJSJobMethods.cancelOneJob (line 53)
  In parallel.job.CJSConcurrentJob/cancelOneJob (line 53)
  In parallel.Job/cancel (line 1242)
  In parallel.Cluster/hDeleteOneJob (line 906)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster reported an error while deleting an unavailable job.  This
job may already have been deleted.] 
[> In parallel.internal.cluster.CJSJobMethods.destroyOneJob (line 77)
  In parallel.job.CJSConcurrentJob/destroyOneJob (line 48)
  In parallel.Job/delete (line 1176)
  In parallel.Cluster/hDeleteOneJob (line 921)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster failed to cancel the job execution. The error was: Unable
to read file
'/network/lustre/iss01/home/zoe.brusselle/.matlab/local_cluster_jobs/R2017b/Job29.in.mat'.
No such file or directory.] 
[> In parallel.internal.cluster.CJSJobMethods.cancelOneJob (line 53)
  In parallel.job.CJSConcurrentJob/cancelOneJob (line 53)
  In parallel.Job/cancel (line 1242)
  In parallel.Cluster/hDeleteOneJob (line 906)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster reported an error while deleting an unavailable job.  This
job may already have been deleted.] 
[> In parallel.internal.cluster.CJSJobMethods.destroyOneJob (line 77)
  In parallel.job.CJSConcurrentJob/destroyOneJob (line 48)
  In parallel.Job/delete (line 1176)
  In parallel.Cluster/hDeleteOneJob (line 921)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster failed to cancel the job execution. The error was: Unable
to read file
'/network/lustre/iss01/home/zoe.brusselle/.matlab/local_cluster_jobs/R2017b/Job30.in.mat'.
No such file or directory.] 
[> In parallel.internal.cluster.CJSJobMethods.cancelOneJob (line 53)
  In parallel.job.CJSConcurrentJob/cancelOneJob (line 53)
  In parallel.Job/cancel (line 1242)
  In parallel.Cluster/hDeleteOneJob (line 906)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster reported an error while deleting an unavailable job.  This
job may already have been deleted.] 
[> In parallel.internal.cluster.CJSJobMethods.destroyOneJob (line 77)
  In parallel.job.CJSConcurrentJob/destroyOneJob (line 48)
  In parallel.Job/delete (line 1176)
  In parallel.Cluster/hDeleteOneJob (line 921)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster failed to cancel the job execution. The error was: Unable
to read file
'/network/lustre/iss01/home/zoe.brusselle/.matlab/local_cluster_jobs/R2017b/Job33.in.mat'.
No such file or directory.] 
[> In parallel.internal.cluster.CJSJobMethods.cancelOneJob (line 53)
  In parallel.job.CJSConcurrentJob/cancelOneJob (line 53)
  In parallel.Job/cancel (line 1242)
  In parallel.Cluster/hDeleteOneJob (line 906)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster reported an error while deleting an unavailable job.  This
job may already have been deleted.] 
[> In parallel.internal.cluster.CJSJobMethods.destroyOneJob (line 77)
  In parallel.job.CJSConcurrentJob/destroyOneJob (line 48)
  In parallel.Job/delete (line 1176)
  In parallel.Cluster/hDeleteOneJob (line 921)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster failed to cancel the job execution. The error was: Unable
to read file
'/network/lustre/iss01/home/zoe.brusselle/.matlab/local_cluster_jobs/R2017b/Job34.in.mat'.
No such file or directory.] 
[> In parallel.internal.cluster.CJSJobMethods.cancelOneJob (line 53)
  In parallel.job.CJSConcurrentJob/cancelOneJob (line 53)
  In parallel.Job/cancel (line 1242)
  In parallel.Cluster/hDeleteOneJob (line 906)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster reported an error while deleting an unavailable job.  This
job may already have been deleted.] 
[> In parallel.internal.cluster.CJSJobMethods.destroyOneJob (line 77)
  In parallel.job.CJSConcurrentJob/destroyOneJob (line 48)
  In parallel.Job/delete (line 1176)
  In parallel.Cluster/hDeleteOneJob (line 921)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster failed to cancel the job execution. The error was: Unable
to read file
'/network/lustre/iss01/home/zoe.brusselle/.matlab/local_cluster_jobs/R2017b/Job35.in.mat'.
No such file or directory.] 
[> In parallel.internal.cluster.CJSJobMethods.cancelOneJob (line 53)
  In parallel.job.CJSConcurrentJob/cancelOneJob (line 53)
  In parallel.Job/cancel (line 1242)
  In parallel.Cluster/hDeleteOneJob (line 906)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
[Warning: The cluster reported an error while deleting an unavailable job.  This
job may already have been deleted.] 
[> In parallel.internal.cluster.CJSJobMethods.destroyOneJob (line 77)
  In parallel.job.CJSConcurrentJob/destroyOneJob (line 48)
  In parallel.Job/delete (line 1176)
  In parallel.Cluster/hDeleteOneJob (line 921)
  In parallel.internal.pool.InteractiveClient>iDeleteJobs (line 869)
  In parallel.internal.pool.InteractiveClient/pRemoveOldJobs (line 481)
  In parallel.internal.pool.InteractiveClient/start (line 315)
  In parallel.Pool>iStartClient (line 772)
  In parallel.Pool.hBuildPool (line 586)
  In parallel.internal.pool.doParpool (line 18)
  In parpool (line 98)] 
connected to 8 workers.
>> >> >> >> >> >> >> >> Run Launcher 24
>> RUN p24_condNP_Invalid launcher11
{Error using run (line 55)
Transparency violation error.
 See Parallel Computing Toolbox documentation about Transparency
} 
>> >> >> Run Second_pass_go_local 24
>> [Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher1.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher10.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher11.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher12.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher13.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher14.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher15.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher16.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher17.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher18.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher19.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher2.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher20.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher21.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher22.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher23.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher24.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher25.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher26.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher27.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher28.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher29.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher3.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher30.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher31.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher32.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher33.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher34.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher35.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher36.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher37.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher38.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher39.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher4.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher40.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher41.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher42.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher43.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher44.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher45.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher46.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher47.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher48.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher49.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher5.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher50.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher6.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher7.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher8.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Valid/res/launcher9.m>@(input)stat_model_go(input,'p24_condP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher1.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher10.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher11.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher12.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher13.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher14.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher15.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher16.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher17.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher18.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher19.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher2.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher20.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher21.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher22.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher23.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher24.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher25.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher26.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher27.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher28.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher29.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher3.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher30.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher31.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher32.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher33.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher34.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher35.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher36.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher37.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher38.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher39.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher4.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher40.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher41.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher42.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher43.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher44.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher45.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher46.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher47.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher48.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher49.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher5.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher50.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher6.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher7.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher8.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condP_Invalid/res/launcher9.m>@(input)stat_model_go(input,'p24_condP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher1.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher10.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher11.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher12.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher13.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher14.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher15.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher16.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher17.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher18.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher19.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher2.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher20.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher21.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher22.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher23.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher24.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher25.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher26.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher27.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher28.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher29.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher3.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher30.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher31.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher32.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher33.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher34.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher35.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher36.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher37.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher38.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher39.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher4.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher40.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher41.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher42.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher43.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher44.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher45.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher46.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher47.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher48.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher49.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher5.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher50.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher6.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher7.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher8.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Valid/res/launcher9.m>@(input)stat_model_go(input,'p24_condNP_Valid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Invalid/res/launcher1.m>@(input)stat_model_go(input,'p24_condNP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Invalid/res/launcher10.m>@(input)stat_model_go(input,'p24_condNP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Invalid/res/launcher2.m>@(input)stat_model_go(input,'p24_condNP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Invalid/res/launcher3.m>@(input)stat_model_go(input,'p24_condNP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Invalid/res/launcher4.m>@(input)stat_model_go(input,'p24_condNP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Invalid/res/launcher5.m>@(input)stat_model_go(input,'p24_condNP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Invalid/res/launcher6.m>@(input)stat_model_go(input,'p24_condNP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Invalid/res/launcher7.m>@(input)stat_model_go(input,'p24_condNP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Invalid/res/launcher8.m>@(input)stat_model_go(input,'p24_condNP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
[Warning: Could not find appropriate function on path loading function handle
/network/lustre/iss01/home/zoe.brusselle/fit/p24_condNP_Invalid/res/launcher9.m>@(input)stat_model_go(input,'p24_condNP_Invalid')] 
[> In Second_pass_go_local (line 13)] 
>> >> >> >> Run Launcher 24
>> RUN p24_condP_Valid launcher_2_1
RUN p24_condP_Invalid launcher_2_1
{Error using run (line 55)
Transparency violation error.
 See Parallel Computing Toolbox documentation about Transparency
} 
>> >> >> Get best Param p 24
>> >> >> >> >>   message with properties:

    Identifier: 'MATLAB:connector:connector:ConnectorNotRunning'
     Arguments: {}

Thu Oct  3 09:31:28 CEST 2019
[34m--- Analysis completed ---[0m
